app/models/explainer.rb

```diff
@@ -61,7 +61,7 @@ def self.update_paragraphs_in_alegre(id, previous_paragraphs_count, timestamp)
 
     base_context = {
       type: 'explainer',
-      team: explainer.team.slug,
+      team: explainer.team.id,
```

Comment:
I believe this needs to be `team_id` instead?

Reply:
In this case, both should work, but `explainer.team_id` is "faster".

Reply:
Thank you both, I updated this to `team_id`